hack/pre-release.sh

```diff
@@ -0,0 +1,207 @@
+
+  echo "Updating the Makefile to version $updated_version"
+  # Example: matches *WMCO_VERSION ?= 9.0.0*
+  sed -i "s/WMCO_VERSION ?= $version/WMCO_VERSION ?= $updated_version/g" Makefile
+
+  echo "Updating the skipRange version in $cluster_service_version_yaml to $updated_version"
+  # Example: matches olm.skipRange: '>=8.0.0 *<9.0.0*'
+  sed -i "s/<$version/<$updated_version/g" $cluster_service_version_yaml
+
+  echo "Updating previous skipRange version in $cluster_service_version_yaml to $version"
+  # Example: matches olm.skipRange: '*>=8.0.0* <9.0.0'
+  sed -i "s/>=$last_version/>=$version/g" $cluster_service_version_yaml
+ 
+  echo "Updating bundle/windows-machine-config-operator.package.yaml to $updated_version"
+  # Example: matches   - currentCSV: windows-machine-*config-operator.v9.0.0*
+  sed -i "s/config-operator.v$version/config-operator.v$updated_version/g" bundle/windows-machine-config-operator.package.yaml
+   
+}
+
+update_OCP_version() { 
+  xy_stream=$1
+  
+  cluster_service_version_yaml=config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml
+  
+  echo "OCP version $version detected"
+
+  # Increment the version by using the stream_var variable to select which column to increment 
+  updated_version=$(echo $version |
+    awk -F. -v OFS=. -v stream_var="$stream_var" '{$stream_var += 1 ; print}')
+
+  # Find the last version in the dockerfiles
+  last_version=$(sed -n 's/.*\([0-9]\.[0-9][0-9]\).*/\1/p' build/Dockerfile)
+
+  echo "Updating $cluster_service_version_yaml version to $updated_version"
+  # Example: matches on any instance of the OCP version, such as 4.14 
+  sed -i "s/$version/$updated_version/g" $cluster_service_version_yaml
+
+  if [[ $CVP == "true" ]]; then  
+    echo "Updating bundle.Dockerfile version to $updated_version"
+    # Example: matches LABEL com.redhat.openshift.versions="*=v4.13*"
+    sed -i "s/=v$version/=v$updated_version/g" bundle.Dockerfile
+
+    echo "Updating bundle.Dockerfile last version to $version"
+    # Example: matches LABEL com.redhat.openshift.versions="=v4.13"
+    sed -i "s/=v$last_version/=v$version/g" bundle.Dockerfile
+
+    echo "Updating bundle/metadata/annotations.yaml version to $updated_version"
+    # Example: matches com.redhat.openshift.versions: "*=v4.13*"
+    sed -i "s/=v$version/=v$updated_version/g" bundle/metadata/annotations.yaml 
+  else
+    echo ""
+    echo "skipping bundle.Dockerfile and bundle/metadata/annotations.yaml due to CVP being a version behind"
+    echo "if these do need to be bumped, rerun with the -cvp flag from your original branch"
+    echo ""
+  fi 
+  
+  if [[ $BASEIMAGE == "true" ]]; then  
+    echo "Updating build/Dockerfile version to $updated_version"
+    # Example: matches FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.19-*openshift-4.13* as build
+    sed -i "s/openshift-$version/openshift-$updated_version/g" build/Dockerfile
```

Comment:
Nit: I suggest making this into a helper function and have a for loop here going over the set of files.